src/core/Slider/Slider.tsx

```diff
@@ -0,0 +1,463 @@
+        onChange?.(newValues);
+      },
+      [currentValues, onChange],
+    );
+
+    const onThumbActivated = React.useCallback((index: number) => {
+      setActiveThumbIndex(index);
+    }, []);
+
+    const handlePointerUp = React.useCallback(
+      (event: PointerEvent) => {
+        updateThumbValue(event, onChange);
+        setActiveThumbIndex(undefined);
+        event.preventDefault();
+        event.stopPropagation();
+      },
+      [onChange, updateThumbValue],
+    );
+
+    const handlePointerDownOnSlider = React.useCallback(
+      (event: React.PointerEvent) => {
+        if (containerRef.current) {
+          const percent = getPercentageOfRectangle(
+            containerRef.current.getBoundingClientRect(),
+            event.pageX,
+          );
+          let pointerValue = min + (max - min) * percent;
+          pointerValue = roundValueToClosestStep(pointerValue, step, min);
+
+          const closestValueIndex = getClosestValueIndex(
+            currentValues,
+            pointerValue,
+          );
+          const [minVal, maxVal] = getAllowableThumbRange(closestValueIndex);
+          pointerValue = getBoundedValue(pointerValue, minVal, maxVal);
+          if (pointerValue === currentValues[closestValueIndex]) {
+            return;
+          }
+          const newValues = [...currentValues];
+          newValues[closestValueIndex] = pointerValue;
+          setCurrentValues(newValues);
+          onChange?.(newValues);
+        }
+      },
+      [min, max, step, currentValues, getAllowableThumbRange, onChange],
+    );
+
+    const pointerMoveFunctionRef = React.useRef<
+      (event: PointerEvent) => void
+    >();
+    React.useEffect(() => {
+      const ownerDoc = containerRef.current?.ownerDocument;
+      if (pointerMoveFunctionRef.current && ownerDoc) {
+        ownerDoc.removeEventListener(
+          'pointermove',
+          pointerMoveFunctionRef.current,
+        );
+      }
+      pointerMoveFunctionRef.current = handlePointerMove;
+    }, [handlePointerMove]);
+
+    const pointerUpFunctionRef = React.useRef<(event: PointerEvent) => void>();
+    React.useEffect(() => {
+      const ownerDoc = containerRef.current?.ownerDocument;
+      if (pointerUpFunctionRef.current && ownerDoc) {
+        ownerDoc.removeEventListener('pointerup', pointerUpFunctionRef.current);
+      }
+      pointerUpFunctionRef.current = handlePointerUp;
+    }, [handlePointerUp]);
+
+    React.useEffect(() => {
+      const ownerDoc = containerRef.current?.ownerDocument;
+      ownerDoc?.addEventListener('pointermove', handlePointerMove);
+      ownerDoc?.addEventListener('pointerup', handlePointerUp);
+      return () => {
+        ownerDoc?.removeEventListener('pointermove', handlePointerMove);
+        ownerDoc?.removeEventListener('pointerup', handlePointerUp);
+      };
+    }, [handlePointerMove, handlePointerUp]);
+
+    return (
+      <div
+        className={cx(
+          'iui-slider-component-container',
```

Comment:
We also need to test on a touch device.

Reply:
I simulated mobile device in devtools and found that I need to add `touch-action: pan-y` for the slider to work.
![touch](https://user-images.githubusercontent.com/9084735/125524427-8cf484be-fb75-428e-a2e2-2601eb0750a1.gif)

Reply:
Are you going to update this in iTwinUI? Do you want me to add it manually for until css is updated?

Reply:
I can update the css but it would be nice if we can collect all changes needed for touch devices.

Reply:
`touch-action: pan-y` is now available in 0.24.1 but we should still test on an actual touch device and see if we need something else